frontend/awx/administration/execution-environments/ExecutionEnvironmentForm.tsx

```diff
@@ -217,11 +217,19 @@ function ExecutionEnvironmentInputs(props: {
         label={t('Description')}
         placeholder={t('Enter a description')}
         isDisabled={props?.executionEnv?.managed || false}
+        helperText={t(
+          'Leave this field blank to make the execution environment globally available.'
```

Comment:
Maybe a mistake? Description shouldn't have this

Reply:
agreed, thanks for catching that. it should be fixed now!